views/docs/history.html

```diff
@@ -27,13 +27,13 @@
   {{#each doc.releases}}
     <tr>
       <td>
-          <a href="https://github.com/electron/electron/blob/{{this.tag_name}}{{../doc.href}}.md">
+          <a href="https://github.com/electron/electron/blob/{{this.data.tag_name}}{{../doc.href}}.md">
             <span class="octicon octicon-mark-github mr-1"></span>
             {{../doc.slug}}.md @ {{this.data.version}}
           </a>
       </td>
       <td>
-          <time data-date="{{this.data.created_at}}" data-format="%B %d, %Y">{{this.created_at}}</time>
+          <time data-date="{{this.data.created_at}}" data-format="%B %d, %Y"></time>
```

Comment:
Thoughts about changing this to `published_at`? I noticed a few cases where they don't match and the `published_at` seems to be the correct date.

```suggestion
          <time data-date="{{this.data.published_at}}" data-format="%B %d, %Y"></time>
```